tests/unit/test_contrib/test_pydantic/conftest.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+import pydantic
+import pytest
+from pydantic import v1 as pydantic_v1
+from pytest import FixtureRequest
+
+
+@pytest.fixture(params=["v1", "v2"])
+def pydantic_version(request: FixtureRequest) -> str:
+    return request.param  # type: ignore[no-any-return]
+
+
+@pytest.fixture()
+def base_model(pydantic_version: str) -> type[pydantic.BaseModel | pydantic_v1.BaseModel]:
```

Comment:
Not sure if doing it like this is the best way or if we want to write out all tests explicitly for v1 and v2. 
I now used a mix of both; Separated tests where the body of the test diverges and parametrised tests where just the setup is different, as long as the setup doesn't get to messy.

@litestar-org/members?

Reply:
sounds reasonable to me :+1: